eppo_client/version.py

```diff
@@ -1 +1 @@
-__version__ = "3.1.4"
+__version__ = "3.2.0"
```

Comment:
_technically_ this interface change is a non-backward compatible breaking change, but fine doing the 3.2 thing

Reply:
Yup my thought exactly